packages/p2p/src/components/app-content.jsx

```diff
@@ -13,18 +13,40 @@ import { useModalManagerContext } from 'Components/modal-manager/modal-manager-c
 import TemporarilyBarredHint from 'Components/temporarily-barred-hint';
 import { buy_sell } from 'Constants/buy-sell';
 import { useStores } from 'Stores';
-
+import { getHoursDifference } from 'Utils/date-time';
 import { localize } from './i18next';
 
 const AppContent = ({ order_id }) => {
     const { buy_sell_store, general_store } = useStores();
     const { showModal, hideModal } = useModalManagerContext();
+    let timeout;
     const {
         notifications: { setP2POrderProps },
+        client: { loginid },
     } = useStore();
     const notification_count = useP2PNotificationCount();
     const history = useHistory();
 
+    const handleDisclaimerTimeout = time_lapsed => {
+        timeout = setTimeout(() => {
+            showModal({ key: 'DisclaimerModal', props: { handleDisclaimerTimeout } });
+            // Display the disclaimer modal again after 24 hours
+        }, (24 - time_lapsed) * 3600000);
+    };
+
+    React.useEffect(() => {
+        const time_lapsed = getHoursDifference(localStorage.getItem(`${loginid}_disclaimer_shown`));
```

Comment:
can we prefix p2p in the key so it's easier to filter and understand that this is only used in p2p